cartography/data/jobs/cleanup/aws_account_dns_cleanup.json

```diff
@@ -6,7 +6,7 @@
     "iterationsize": 100
   },
   {
-    "query": "MATCH ()<-[r:DNS_POINTS_TO]-(:AWSDNSRecord)-[:MEMBER_OF_DNS_ZONE]->(:AWSDNSZone)<-[:RESOURCE]-(:AWSAccount{id: {AWS_ID}}) WHERE r.lastupdated <> {UPDATE_TAG} WITH r LIMIT {LIMIT_SIZE} DETACH DELETE (r) return COUNT(*) as TotalCompleted",
+    "query": "MATCH (:ESDomain:Ip:NameServer:EC2Instance:AWSDNSRecord:DNSRecord)<-[r:DNS_POINTS_TO]-(:AWSDNSRecord)-[:MEMBER_OF_DNS_ZONE]->(:AWSDNSZone)<-[:RESOURCE]-(:AWSAccount{id: {AWS_ID}}) WHERE r.lastupdated <> {UPDATE_TAG} WITH r LIMIT {LIMIT_SIZE} DETACH DELETE (r) return COUNT(*) as TotalCompleted",
```

Comment:
Doesn't look scalable. For example, do we know how many labels can be expressed that way in Cypher Query? Another way could be to use WHERE labels(n) in []. Either way, I question the need to be explicitly define labels here and possibly  move the logic on when this cleanup occurs.

Reply:
fix in updated code and cleanup is broken out per label